recipe/meta.yaml

```
@@ -8,10 +8,10 @@ package:
 source:
   - path: ./build_base.sh
   - url: https://github.com/openforcefield/openff-toolkit/archive/{{ version }}.tar.gz
-    sha256: 5bea41fdf5af5ffec8ca5203b714d0c7359507762df919cd42d1fec3b2de1686
+    sha256: b15aec2b0d4ae6384d96a6b53178a22be95ac39760b3a534ba740463eb819a27
```

Get this by running `curl -sL https://github.com/openforcefield/openff-toolkit/archive/refs/heads/examples_helper.zip | openssl sha256`

```suggestion
    sha256: 9e7b43a3e79760b3293c5ddb6556feb4196025e1f3325c3866cdc997c2d6b073
```